wsit/ws-sx/wssx-impl/src/main/java/com/sun/xml/ws/security/trust/impl/IssueSamlTokenContractImpl.java

```diff
@@ -175,7 +175,7 @@ private EncryptedKey encryptKey(final Document doc, final byte[] encryptedKey, f
             cipher.init(XMLCipher.WRAP_MODE, pubKey);
 
             encKey = cipher.encryptKey(doc, new SecretKeySpec(encryptedKey, "AES"));
-            final KeyInfo keyinfo = new KeyInfo(doc);
+            final KeyInfo keyinfo = new KeyInfo(doc);//tady
```

Comment:
Did you forget to remove this comment or it is intentional?

Reply:
whoops :-), fixed